### PR TITLE
build(semantic-release): add @semantic-release/git plugin

### PR DESCRIPTION
**Pull request by leandrogrillo**
Not sure if this will work. Maybe we'll need to setup permission to @leroy-robot commit in `master`.

We'll need to test it.

Ps: add some restriction to `master` and `develop`. 
@andreolvr https://github.com/leroy-merlin-br/backyard/settings/branches

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,17 @@
   "publishConfig": {
     "access": "public"
   },
+  "release": {
+    "plugins": [
+      [
+        "@semantic-release/git",
+        {
+          "assets": "package.json",
+          "message": "chore(release): bump package to ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
```
Comment (leandrogrillo) [inline package.json]: @andreolvr is this message good?

Comment (andreolvr) [reply package.json]: seems good to me.

Comment (andreolvr) [inline package.json]:
```suggestion
          "message": "build(npm): bump package version to ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
```

Comment (andreolvr) [reply package.json]: I just created a suggestion about this.

**Pull request merged**